azure-service-broker/README.md

```diff
@@ -0,0 +1,336 @@
+
+**Table of Contents**
+
+* [Prerequisites](#prerequisites)
+* [Azure Quickstart](#azure-setup)
+    * [Configure your Azure account](#configure-your-azure-account)
+    * [Create a Resource Group](#create-a-resource-group-for-aks)
+    * [Create a service principal](#create-a-service-principal)
+* [Deploying the Broker](#deploying-the-broker)
+    * [Cloning](#cloning)
+    * [Configuring](#configuring)
+    * [Pushing]()
+    * [Registering]()
+        * [Viewing the Marketplace](#viewing-the-marketplace)
+* [Automation](#automation)
+    * [Working Assumptions](#working-assumptions)
+    * [Automating Deployment](#automating-deployment)
+        * [Usage](#usage)
+        * [CI Pipelines](#ci-pipelines)
+        * [Using Concourse.ci](#using-concourse.ci)
+       * [Using terraform CLI](#using-terraform-cli)
+    * [Troubleshooting](#troubleshooting)
+    
+
+## Prerequisites
+
+In order to complete the tutorial, please be sure you have:
+
+* A [Microsoft Azure account](https://portal.azure.com).
+* A working knowledge of **Azure [CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)** or use of the [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview?view=azure-cli-latest)
+    * _Optionally, you use the official [`azure-cli`](https://hub.docker.com/r/microsoft/azure-cli/dockerfile) container_
+* A working knowledge of **Cloud Foundry** with experience using the [CLI](https://docs.cloudfoundry.org/cf-cli/).
+* A working knowledge of space-scoped service brokers. For an  introduction, see the [Simple Service Broker Quickstart Tutorial](../simple-service-broker).
+* A Cloud Foundry account and a space to deploy apps.
+
+## Azure Setup
+
+### Configure your Azure account
+
+> First let's identify your Azure subscription and save its output for later use in our deployment
+
+1. Run `az login` and follow the instructions in the command output to authorize `az` to use your account
+1. List your Azure subscriptions:
+    ```console
+    az account list -o table
+    ```
+1. Copy your subscription ID and save it in an environment variable:
+
+    **Bash**
+    ```console
+    export AZURE_SUBSCRIPTION_ID="<SubscriptionId>"
+    ```
+
+    **PowerShell**
+    ```console
+    $env:AZURE_SUBSCRIPTION_ID = "<SubscriptionId>"
+    ```
+
+### Create a Resource Group
+
+Create one with the az cli using the following command.
+
+```console
+az group create --name <CHANGEME> --location eastus
+```
+### Create Redis Cache Store
+
+Open Service Broker for Azure uses Redis as a backing store for its state. We recommend using a managed Redis service, such as Azure Redis Cache. You can use the Azure CLI to determine if Azure Redis Cache is enabled for your subscription:
+
+```sh
```

Comment:
You should show commands in either only bash or bash + PS, but not some of each. I would prioritize Powershell over bash because that's going to be much more common in govt.